WRITING.md

```diff
@@ -1,238 +1,127 @@
 # Solid Docs Writing Guide
 
-_Read the [Solid Docs Manifesto](https://github.com/solidjs/solid-docs-next#the-solid-docs-manifesto) first!_
+## Introduction
 
-_This is a draft! Help us improve it by making an issue or [reaching out directly](https://twitter.com/jutanium)._
+Thank you for your interest in writing for Solid.
+This writing guide is in-progress, so please be sure to check it often as we adjust things.
+We welcome suggestions and feedback.
 
-This document explains how to write content for the new Solid docs.
+Bear in mind that the following guide is a general guideline, but we will use it as a framework to edit any incoming contributions.
+This is to ensure to that Solid's documents remain consistent in tone, voice, structure, and conventions.
 
-We start broadly: how do we plan our content to most effectively teach the reader?
+## Visual style
 
-Then, we narrow in on the paragraph and sentence: how should the voice of the content _feel_, and how do we make sure the language is accessible?
+Solid wants to maintain a consistent brand across all of its documentation.
+As such, we have several guidelines that will dictate how your contributions will look once published.
+While it would be great if you followed these exactly — and our contributing editors would sure love you for it — we know that you'd probably prefer to focus on getting great Solid content ready to go.
+So, please try to follow this visual framework as best as you can to make the process as smooth as possible.
 
-Lastly, we discuss the specific terminology to use when referring to Solid concepts.
+All that said, visual styling is very important for a product like Solid.
+We want to present a consistent and professional face to our users and partners.
 
-**Contents**
+Of course, trends in documentation styling change over time, so we're fully aware that these guidelines will likely shift as we consider adopting new best practices.
 
-- [When do I need to follow this?](#when-do-i-need-to-follow-this)
-- [Writing to teach](#writing-to-teach)
-- [Content types](#content-types)
-- [Tone and style](#tone-and-style)
-- [Accessibility](#accessibility)
-- [Terminology](#terminology)
+### Layout
 
-## When do I need to follow this?
+### Code usage examples
```

Comment:
[Nitpick] I think naming it Syntax would be helpful :) I also provided areas where you can put examples. 
```suggestion
### Syntax 
- Example #1 
- Example # 2
```